### DISCUSSION TITLE
HERO10 1.20.70 needs enhancement to support command !MHSTO

### DISCUSSION THREAD

**Discussion by AnthonyBassOrDeath** [gopro/labs#400] [Feature Requests]
Really need a firmware based on 1.20.70, adding enhancement to support command !MHSTO to controll the length of the Hindsight timeout.

Below is the background of this request.
It's a popular issue that HERO10 hindsight gets glitching/corrupted videos, the only workaround so far is to use 1.20 or 1.20.70 firmware. But 1.20.70 does NOT support command !MHSTO.

I believe this combination labs firmware would be very helpful to the anglers. Much Much Much thanks if someone would help this out.

_Here is hindsight issue testing performed by [sweetstar12545](https://community.gopro.com/s/profile/0053b00000DApakAAD) 
[93607691](https://www.edcint.co.nz/tmp/gopro/history.txt)

And here is the topic disscussion: https://community.gopro.com/s/question/0D53b00008cQcYWCA0/why-are-many-of-my-hero10-black-hindsight-video-captures-glitching?language=en_US_

**Comment by dnewman-gpsw** [comment]
Labs is based on current firmware (now 1.50.70), there will be not releases back on older firmware.

**Reply by AnthonyBassOrDeath** [reply]
Thanks for reply. Unfortunately 1.50.70 has fatal bug of hindsight same as descriped above, I've tried this version.

If it is the policy or rule of GoPro Labs, my request transforms to work out the hindsight bug fix on the latest firmware version, but it's been never updated to the users of HERO10, none of plan or status, though lots of them asked about it. This really upset people.

**Reply by dnewman-gpsw** [reply]
I see on the GoPro community page link, a GoPro Support person was following up 6 days ago with a ticket number(s) to address.  I would suggest continuing to follow up with support, to help them reproduce it.  If the bug is in the main firmware, Labs will have it too.  Labs can only be built upon current firmware for QA reasons, and this is the reason we can't implement your request. 

P.S. I did try this, add lots a camera shake, in 10 captures no failures, I was using SanDisk Extreme Pro 400GB media.

**Reply by AnthonyBassOrDeath** [reply]
Thank you sir~ I'll follow that page for the update. 
Actually I heard some people get the replaced GoPro from the support team, but the HindSight issue remains.

BTW, to reproduce the bug is simple: 
1 Settings: rear screen off time 5 minutes, hindsight time 30 secs;
2 Start hindsight, wait till the rear screen turns off, then press the recording button, 1 or 2 secs later press the button again;

With my HERO10, 1.50.70, this scenario makes lots of corrupted videos. Tried qualified media Lexar and Samsun which are  in the GoPro official supporting media list, makes no difference.

**Reply by dnewman-gpsw** [reply]
I was now be able to reproduce the error, and have documented the bug. Hopefully it can be addressed in a future update.

**Reply by AnthonyBassOrDeath** [reply]
Cheers~. Thank you sir, have a nice day.

**Reply by AnthonyBassOrDeath** [reply]
Hello sir, sorry to bother again. But there were no update from the official product support page nor the GoPro community page link mentioned above. Just wondering if Black10 firmware update service is not supported anymore. Thanks~

**Reply by dnewman-gpsw** [reply]
There should be an update later in the year.

**Reply by AnthonyBassOrDeath** [reply]
@dnewman-gpsw Hello David, happy new year~  Just wondering if come out fix version. Thanks~

**Reply by dnewman-gpsw** [reply]
The bug is still open, but it is not something Labs can fix.

**Reply by AnthonyBassOrDeath** [reply]
Understood. Could you pass me the product manager/project manager contact info like email/whatsapp please ? I just want to talk with them on how to fix or walk around this bug. For instance release another non-labs version with default 20 hours hindsight timeout parameter. Really just don't want to give up. Thank you. @dnewman-gpsw

**Reply by dnewman-gpsw** [reply]
The bug has been filed, visible to whoever the product manager is.  There nothing more you can contribute to get it fixed.  Please keep discussions focused to Labs on this forum, it is not a substitute for general tech support, or for GoPro personnel info requests.

**Comment by YertyArg** [comment]
El Lab... para Gopro 10 no será actualizado? La última versión es de enero de 2023!!!!

**Reply by dnewman-gpsw** [reply]
Labs updates after the main firmware.  Likely soon
